projects/application/DatSphere-en.md

```diff
@@ -0,0 +1,62 @@
+# Project Name DatSphere
+
+## Category 
+application
+
+## Project Description
+An open-source, decentralized drive service built on ipfs, filecoin
```

Comment:
A more detailed description will be appreciated since it will help our community to know more about your project and try it out.

Reply:
Thanks Longfei, we will be working on this and will soon get back to you